WordPress/src/main/res/values/styles.xml

```diff
@@ -223,14 +221,14 @@
         <item name="colorButtonNormal">@color/white</item>
     </style>
 
-    <style name="WordPress.BlueButton" parent="Widget.AppCompat.Button.Colored">
-        <item name="colorButtonNormal">@color/blue_wordpress</item>
-        <item name="android:textColor">@color/nux_primary_button</item>
+    <style name="WordPress.PrimaryButton" parent="Widget.AppCompat.Button.Colored">
+        <item name="colorButtonNormal">@color/blue_medium</item>
+        <item name="android:textColor">@color/white</item>
     </style>
 
-    <style name="WordPress.WhiteButton" parent="Widget.AppCompat.Button.Colored">
+    <style name="WordPress.StandardButton" parent="Widget.AppCompat.Button.Colored">
         <item name="colorButtonNormal">@color/white</item>
-        <item name="android:textColor">@color/blue_wordpress</item>
+        <item name="android:textColor">@color/grey_dark</item>
     </style>
```

Comment:
We had a brief discussion (p1500397426988031-slack-mobile-login) and here's the agreed suggestion:

* `WordPress.Button` for the default version, currently white.
* `WordPress.Button.Primary` for the primary button on the page, currently blue.

Rationale:

1. The simple `Button` naming has an implication of being default, which is. That is the style we want to use as our default, reference button on Android.
2. `Primary` is a variation on the base button, specifically one that should be used once for each screen, as it's designed to be "the primary action the user would take on the visible screen".
3. Their value is semantic, it refer to the meaning of the button, it's not styling, which instead is defined inside the spec.
4. Side effect: it maps Calypso.

We also highlighted two details that are relevant:

(a) There might be ambiguity of `Primary` between this semantic use, and Android themes usage for _primary color_.
(b) We might want to discuss if we can remove the `Gray` variation of the default button from the design spec. This is a separate discussion that doesn't block the new login PRs as this variation isn't used anywhere.